package.json

```diff
@@ -19,7 +19,7 @@
     "common-tags": "^1.8.0",
     "dd-botkit": "^1.0.9-1",
     "pg": "^7.4.3",
-    "typeorm": "^0.2.7"
+    "url-unshort": "^5.0.0"
```

Comment:
Still shouldn't use other dependencies when there are better ways to unshorten links.